tasks.php

```diff
@@ -166,117 +166,117 @@
     18 => "In Progress",
 ];
 asort($tasks_status_array);
-$os_array = [
+const TASK_OS = [
     0 => "All",
-    1 => "Windows 3.1",
-    2 => "Windows 95",
-    3 => "Windows 98",
-    4 => "Windows ME",
-    5 => "Windows 2000",
-    6 => "Windows NT",
-    7 => "Windows XP",
+    37 => "Android",
+    21 => "BeOS",
+    17 => "BSDI",
+    18 => "FreeBSD",
+    22 => "HP-UX",
+    23 => "IRIX",
+    35 => "iOS",
+    36 => "iPadOS",
+    16 => "Linux",
+    15 => "Mac OS X / macOS",
     8 => "Mac System 7",
     9 => "Mac System 7.5",
     10 => "Mac System 7.6.1",
     11 => "Mac System 8.0",
     12 => "Mac System 8.5",
     13 => "Mac System 8.6",
     14 => "Mac System 9.x",
-    15 => "Mac OS X / macOS",
-    16 => "Linux",
-    17 => "BSDI",
-    18 => "FreeBSD",
     19 => "NetBSD",
-    20 => "OpenBSD",
-    21 => "BeOS",
-    22 => "HP-UX",
-    23 => "IRIX",
     24 => "Neutrino",
+    20 => "OpenBSD",
     25 => "OpenVMS",
     26 => "OS/2",
     27 => "OSF/1",
+    99 => "Other",
     28 => "Solaris",
     29 => "SunOS",
+    5 => "Windows 2000",
     30 => "Windows 2003",
-    31 => "Windows Vista",
+    1 => "Windows 3.1",
     32 => "Windows 7",
     33 => "Windows 8",
     34 => "Windows 10",
-    35 => "iOS",
-    36 => "iPadOS",
-    37 => "Android",
-    99 => "Other",
+    2 => "Windows 95",
+    3 => "Windows 98",
+    6 => "Windows NT",
+    4 => "Windows ME",
+    31 => "Windows Vista",
+    7 => "Windows XP",
+    // max 37
```

Comment:
39 with `"0 => All"` and `"99 => Other"`

Reply:
But the highest key value is 37, this makes sense to me.